### ISSUE TITLE
CLARKE exosuits aren't resistant against EMP and therefore aren't useful for working with the singularity or most of engineering (their intended purpose) 

### ISSUE THREAD

**Issue by githubuser4141** [d3athrow/vgstation13#28107]
#### Description of issue:

CLARKE exosuits right now don't have much of a purpose, since most engineering tasks can be accomplished just fine as a person, and you can't interact with pipes or computer consoles in a CLARKE, so there's really no use. I was thinking that if they were to be made EMP immune (or at least heavily resistant, so they don't die in one hit) they'd be useful for checking on the singulo while it's active or containing loose singulos by flying near them and dropping a BoH/antimatter jar near them. Currently, CLARKEs have 100 health and no laser resistance so they'd still die in 4 shots to a standard laser shots. 

#### Difference between expected and actual behavior:

I expected a engineering mech to be shielded from EMPs, and not be instantly destroyed when near a singularity that was contained.

#### Steps to reproduce
Step 1: Get into a CLARKE
Step 2: Find a singularity
Step 3: Drop a few antimatter jars near it (and watch as various pieces of your CLARKE break simply by being near it)
Step 4: Scream as your CLARKE is replaced by a pile of wreckage and you're left alone in space
Step 5: Die and become forgotten about as nobody can hear you scream in space

**Comment by Kurfursten** [comment]
Clarke is for transporting engineering goods with the clamp (air pump, scrubbers, canisters of air to refill rooms, welding tank) to wreckage sites, engine materials to engineering (rad collectors, shard, etc) and also just traveling quickly in general without risk of ZAS (I recommend the running board to make easy hop in hop out), and general space repairs/building since they have perfect handling in space without drift or slipping

This doesn't solve your complaint, it just doesn't seem like you knew what its major purpose is. Clarkes are great. Love 'em.

**Comment by Exxion** [comment]
Mechs in general are funny with the singularity because of its ability to unanchor them (as well as nearly anything else)

**Comment by githubuser4141** [comment]
> Clarke is for transporting engineering goods with the clamp (air pump, scrubbers, canisters of air to refill rooms, welding tank) to wreckage sites, engine materials to engineering (rad collectors, shard, etc) and also just traveling quickly in general without risk of ZAS (I recommend the running board to make easy hop in hop out), and general space repairs/building since they have perfect handling in space without drift or slipping
> 
> This doesn't solve your complaint, it just doesn't seem like you knew what its major purpose is. Clarkes are great. Love 'em.

Yeah, not only that but a Clarke with a phazon cell and RED is pretty useful for sealing breaches quickly, and a decent combat mech if your opponent doesn't have a gun. I found it odd that it lacks any capability for containing the singularity since their cargo capacity + inbuilt jetpack + speed make it feel like they'd be the perfect tool for dropping a few jars near the singularity.

**Comment by DamianX** [comment]
I don't think EMP resistance to mechs would be wise, so I really don't know if there's anything to do about this.
